cmd/snap-exec/main_test.go

```diff
@@ -99,14 +104,18 @@ func (s *snapExecSuite) TestFindCommand(c *C) {
 	c.Assert(err, IsNil)
 
 	for _, t := range []struct {
+		app      string
 		cmd      string
 		expected string
 	}{
-		{cmd: "", expected: `run-app cmd-arg1 $SNAP_DATA`},
-		{cmd: "stop", expected: "stop-app"},
-		{cmd: "post-stop", expected: "post-stop-app"},
+		{app: "app", cmd: "", expected: "/snap/snapname/unset/run-app cmd-arg1 $SNAP_DATA"},
```

Comment:
Without mocking distro relase data, this will break on my dev box which is not Ubuntu/Debian, hence snap mount path is `/var/lib/snapd/snap`.

Reply:
Ah, I suspect it doesn't require a mock so much as _not_ hard-coding this, which I never should have done, my apologies. Does the new push work?